File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
 
 int len = nums.length;
 boolean duplicate =false;
 for(int i=0;i<len;i++){
for(int j=i+1;j<len;j++){
if(nums[i]==nums[j]){
return true;
}
}
 }
 return duplicate;
    }
}

File: Data Structures & Algorithms/binary-search/submission-0.kt
class Solution {
    fun search(nums: IntArray, target: Int): Int    {
        if(nums.size==0 || nums==null) return -1

        val len = nums.size
        var left = 0
        var right = len-1

    while(left<=right){
        val mid = left +(right-left)/2
        // check if nums mid is equal to target
        if(nums[mid]==target){
            return mid
        }
       // check if mid is greater than target
        else if(nums[mid]<target){
            left =mid+1
        }
        else{
            right = mid-1
        }
    }
        return -1

    }
}
